data/telemetry.py

```diff
@@ -0,0 +1,172 @@
+import wpilib
```

Comment:
May want to seperate the drive train telem from other telem and name each in accordance to what they do or what robot they support. Could do this in a refactor later

Reply:
Noted, will add to Kanban board